custom_components/localtuya/light.py

```diff
@@ -19,18 +19,36 @@
 )
 
 from .common import LocalTuyaEntity, async_setup_entry
+from .const import CONF_BRIGHTNESS_LOWER, CONF_BRIGHTNESS_UPPER
 
 _LOGGER = logging.getLogger(__name__)
 
 MIN_MIRED = 153
 MAX_MIRED = 370
 
+DEFAULT_LOWER_BRIGHTNESS = 29
+DEFAULT_UPPER_BRIGHTNESS = 1000
+
+
+def map_range(value, from_lower, from_upper, to_lower, to_upper):
+    """Map a value in one range to another."""
+    mapped = (value - from_lower) * (to_upper - to_lower) / (
+        from_upper - from_lower
+    ) + to_lower
+    return int(min(max(mapped, to_lower), to_upper))
```

Comment:
Why not `round` instead of `int`?

Reply:
Only n00bz use `round`. So yeah, let's use `round`!